test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -308,9 +369,44 @@ void test_ucp_ep_reconfig::skip_non_p2p()
     }
 }
 
+bool test_ucp_ep_reconfig::has_bond_iface()
+{
+    auto context = sender().ucph();
+    const ucp_tl_resource_desc_t *rsc;
+
+    ucs_carray_for_each(rsc, context->tl_rscs, context->num_tls) {
+        std::string tl_name(rsc->tl_rsc.dev_name);
+
+        if (tl_name.find("bond") != std::string::npos) {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+void test_ucp_ep_reconfig::ensure_reused_lanes_reconfigurable()
+{
+    if (!reuse_lanes()) {
+        return;
+    }
+
+    if (has_transport("ud_v") || has_transport("ud_x")) {
+        UCS_TEST_SKIP_R("the test requires at least 2 lanes, while UD has only "
+                        "1");

```

Comment:
what does it mean that UD "has" only 1 lane? every transport is creating one lane..

Reply:
We discussed it before and decided to use default num_paths value for the tests.

Reply:
it does not create rma_bw lanes, but anyway this is not required anymore so removed.